cmd/performance-profile-creator/cmd/root.go

```diff
@@ -163,6 +198,7 @@ func init() {
 	rootCmd.PersistentFlags().StringVarP(&args.mustGatherDirPath, "must-gather-dir-path", "M", "must-gather", "Must gather directory path")
 	rootCmd.MarkPersistentFlagRequired("must-gather-dir-path")
 	rootCmd.PersistentFlags().StringVarP(&args.profileName, "profile-name", "N", "performance", "Name of the performance profile to be created")
+	rootCmd.PersistentFlags().StringVarP(&args.tmPolicy, "topology-manager-policy", "Q", "restricted", "Kubelet Topology Manager Policy of the performance profile to be created. [Valid values: single-numa-node, best-effort, restricted, none]")
```

Comment:
why "Q"?

Reply:
Ran out of letter :) We used up T, M and P so thought Q for Kubelet (Qbelet) Topology Manager Policy purely for the phonetic reasons